Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -58,7 +58,7 @@ ref class AppEventListener
 
 extern "C" void _ApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg);
 extern "C" void EbrApplicationActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs^ args);
```

Comment:
>EbrApplicationActivated [](start = 16, length = 23)

nit: Maybe rename these to OnApplicationActivated, OnApplicationLaunched, etc. (drop Ebr)?